protocol_connect.go

```diff
@@ -511,6 +511,21 @@ func (cc *connectUnaryClientConn) validateResponse(response *http.Response) *Err
 		}
 		cc.responseTrailer[strings.TrimPrefix(k, connectUnaryTrailerPrefix)] = v
 	}
+	err := connectValidateUnaryResponseContentType(
```

Comment:
Could this be moved into the validate response function?

Reply:
It is. Do you mean inlined? I had made it a separate function to make it easier to test.

Reply:
Sorry misread, thought the method on the duplexcall was due to it not being part of this function, but thats for the request params.